### DISCUSSION TITLE
How can I show/hide DataGrid columns based on Responsive?

### DISCUSSION THREAD

**Discussion by pjmagee** [Megabit/Blazorise#2672] [Q&A]
I dont see `Display` property like how it exists on normal Grid `Column` type.

**Comment by stsrki** [comment]
I don't think adding Display would have much sense on a Datagrid column. On what part of the grid would be applied? Header cell, body cell, footer cell, or all?

Can you tell us what you want to accomplish so we can understand it better?

**Reply by pjmagee** [reply]
Certain columns in the Table make sense to show on a larger screen, if the grid is reduced to a smaller screen, then basically just hide some columns because they have less context or to provide a more minimal view. I suppose I could just use the normal Table type and do custom Paging and maybe that would be better?

**Reply by stsrki** [reply]
We can consider adding it. It's not hard. In the meantime, as a workaround you can add class names explicitly, eg. `Class="d-sm-none"`.

**Reply by pjmagee** [reply]
Thank you!
